lib/reporters/mini.js

```diff
@@ -117,6 +121,7 @@ MiniReporter.prototype.unhandledError = function (err) {
 MiniReporter.prototype.reportCounts = function (time) {
 	var lines = [
 		this.passCount > 0 ? '\n   ' + colors.pass(this.passCount, 'passed') : '',
+		this.knownFailureCount > 0 ? '\n   ' + colors.error(this.knownFailureCount, 'known failure') : '',
```

Comment:
`failure` should be pluralized, see https://github.com/cgcgbcbc/ava/blob/8d5a0c21583ae12ca5174e78651a0d72de181fbe/lib/reporters/mini.js#L144 for an example.